tests/test-amp-image-dimension-extractor.php

```diff
@@ -2,6 +2,8 @@
 
 define( 'AMP_IMG_DIMENSION_TEST_VALID_FILE', dirname( __FILE__ ) . '/assets/wordpress-logo.png' );
 define( 'AMP_IMG_DIMENSION_TEST_INVALID_FILE', dirname( __FILE__ ) . '/assets/not-exists.png' );
+define( 'IMG_350', 'http://i0.wp.com/placehold.it/350x150.png');
+define( 'IMG_1024', 'http://i0.wp.com/placehold.it/1024x768.png');
```

Comment:
Any way we could reasonably/easily mock the remote fetches so that tests can work offline (and not rely on an external service)?